pkg/util/oc/oc.go

```diff
@@ -78,3 +87,18 @@ func WaitSMCPReady(t test.TestHelper, ns string, name string) {
 	t.T().Helper()
 	shell.Executef(t, `oc -n %s wait --for condition=Ready smcp/%s --timeout 300s`, ns, name)
 }
+
+func AllResourcesDeleted(t test.TestHelper, ns string, checks ...assert.CheckFunc) {
+	t.T().Helper()
+	shell.Execute(t,
+		fmt.Sprintf(`oc get all -n %s`, ns),
```

Comment:
This doesn't return Secrets and some other resources. Did you mean to include Secrets in this check or not?

Reply:
Before we only check pods, now I'm trying to expand the check but after a delete of the namespace nothing will be there

Reply:
Yes, but this is supposed to be called before you delete the namespace. The purpose is to check whether the namespace is empty after deleting the SMCP for example (i.e. whether the operator deletes everything that was created as a result of the SMCP)..